.github/workflows/build.yml

```
@@ -0,0 +1,26 @@
+name: Build
```

```suggestion
name: Deploy
```

(Also rename the file?)